service/http.proxy/src/test/scripts/org/kaazing/gateway/service/http/proxy/http.proxy.PUT.request.rpt

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# client
+#
+
+connect tcp://localhost:8110
+connected
+
+write "GET /get HTTP/1.1\r\n"
+write "Host: localhost:8110\r\n"
+write "User-Agent: Mozilla/5.0\r\n"
+write "Connection: keep-alive\r\n"
+write "Upgrade-Insecure-Requests: 1\r\n"
+write "\r\n"
+
+read status "200" /.*/
```

Comment:
@vstratan Do not combine k3po HTTP scripting language with tcp level scripting. If you look at the log you will notice that some instructions are `failed` even if the test is successful.